docs/components/IllustrationSection.js

```diff
@@ -1,26 +1,18 @@
 // @flow strict
 import { Box, Flex, Heading } from 'gestalt';
 import { type Node } from 'react';
+import IllustrationContainer from './IllustrationContainer.js';
 
 type Props = {|
   title?: string,
   children?: Node,
+  grid?: 'auto-fill' | 'auto-fit',
+  min?: number,
 |};
 
-function HomePageSection({ children, title }: Props): Node {
+function IllustrationSection({ children, title, grid = 'auto-fit', min = 285 }: Props): Node {
```

Comment:
Can you store 285 in a top-of-file const for easy discovery/tweaking in the future?

Reply:
DONE